tests/x11regressions/remote_desktop/persistent_vncsession_xvnc.pm

```diff
@@ -0,0 +1,119 @@
+# Copyright (C) 2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Remote Login: Persistent VNC Session with tigervnc and xvnc
+# Maintainer: Chingkai <qkzhu@suse.com>
+# Tags: tc#1586209
+
+use strict;
+use base 'basetest';
+use testapi;
+use lockapi;
+use utils;
+
+sub start_vncviewer {
+    x11_start_program "vncviewer 10.0.2.1:1 -Fullscreen";
+    if (check_screen 'vnc_certificate_warning', 10) {
+        send_key "ret";
+    }
+    if (check_screen 'vnc_certificate_warning-2', 10) {
+        send_key "ret";
+    }
```

Comment:
better not use `check_screen` with a low timeout value but `assert_screen` on multiple tags and `match_has_tag`